students/chris_delapena/lesson03/list_lab.py

```diff
@@ -62,7 +63,7 @@
 print("***SERIES 3***")
 #Display list
 print(fruit)
-fruit_3 = fruit
+fruit_3 = fruit_1
 #Go through list asking if user likes that fruit
 for j in fruit_3:
```

Comment:
Note when response is 'no' the next fruit in the list is skipped - can you see why?